### PR TITLE
Use the Google Form's spreadsheet as the data source.

### PR DESCRIPTION
**Pull request by isaprykin**
The data is now displayed from the same Google Sheet as the one where the Google Form writes.
The format is also changed since this spreadsheet has more columns.  The column headers are updated to match the spreadsheet.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -34,9 +34,10 @@ Thanks is a page where Igor and Jasmine can log and view messages of graditude t
 
 <table id="thanksTable">
     <tr>
-      <th>Name</th>
-      <th>Date</th>
-      <th>Thanks</th>
+      <th>Timestamp</th>
+      <th>Who do you want to thank?</th>
+      <th>What are you thankful for?</th>
+      <th>Who are you?</th>
```
Comment (sunnyeyez123) [inline index.md]: Can you change this to Thanker

File: index.md [hunk 2]
```diff
@@ -34,9 +34,10 @@ Thanks is a page where Igor and Jasmine can log and view messages of graditude t
 
 <table id="thanksTable">
     <tr>
-      <th>Name</th>
-      <th>Date</th>
-      <th>Thanks</th>
+      <th>Timestamp</th>
```
Comment (sunnyeyez123) [inline index.md]: Can you change this to "Date"

File: index.md [hunk 3]
```diff
@@ -34,9 +34,10 @@ Thanks is a page where Igor and Jasmine can log and view messages of graditude t
...
+      <th>Who do you want to thank?</th>
```
Comment (sunnyeyez123) [inline index.md]: This should be "Thankee"

**Review by sunnyeyez123 — CHANGES_REQUESTED** [review]

**Review by isaprykin — COMMENTED** [review]
Thanks for the review. Please take another look.

**Review by sunnyeyez123 — APPROVED** [review]

**Review by sunnyeyez123 — APPROVED** [review]

**Pull request merged**